src/datePicker/DatePicker.tsx

```diff
@@ -56,6 +57,7 @@ export class DatePicker extends PureComponent<DatePickerProps, DatePickerState>
     isOutsideRange: day => DayPicker.DateUtils.isDayBefore(day, new Date()),
     orientation: DatePickerOrientation.HORIZONTAL,
     fromMonth: new Date(),
+    toMonth: new Date(new Date().getFullYear() + 1, new Date().getMonth(), new Date().getDate()),
```

Comment:
is it possible to update the prop `isOutsideRange` instead? Like that we would have one way to handle the bounds.
something like this:
```
const dateInOneYear = new Date(new Date().getFullYear() + 1, new Date().getMonth(), new Date().getDate())
...
isOutsideRange: day => DayPicker.DateUtils.isDayBefore(day, new Date()) || DayPicker.DateUtils.isDayAfter(day, dateInOneYear)
```
I'm just not sure what the results will be in the UI.

Reply:
I already tried to use `isOutsideRange ` and the result is not what we expected

`toMonth` is responsible of the min and max month we can navigate through (visibility of navigation arrow buttons)
`isOutsideRange` is responsible of the date we can select but even if we could select only the dates in 1 year range we still would be able to navigate in year far away.

In our case `isOutsideRange` is used to prevent selecting past days in current month